docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,152 @@
+==============
+Hudi connector
+==============
+
+.. raw:: html
+
+  <img src="../_static/img/hudi.png" class="connector-logo">
+
+The Hudi connector enables querying `Hudi <https://hudi.apache.org/docs/overview/>`_
+tables synced to Hive metastore. The connector uses the metastore only to track
+partition locations. It makes use of the underlying Hudi filesystem and input formats
+to list data files. To learn more about the design of the connector, please check out
+`RFC-40 <https://github.com/apache/hudi/blob/master/rfc/rfc-40/rfc-40.md>`_.
+
+Requirements
+------------
+
+To use Hudi, we need:
+
+* Network access from the Trino coordinator and workers to the distributed object storage.
+* Access to a Hive metastore service (HMS).
+* Network access from the Trino coordinator to the HMS.
+
+Configuration
+-------------
+
+The connector requires a Hive metastore for table metadata and supports the same
+metastore configuration properties as the :doc:`Hive connector
+</connector/hive>`. At a minimum, ``hive.metastore.uri`` must be configured.
+
+The connector recognizes Hudi tables synced to the metastore by the Hudi sync tool.
+If non-Hudi tables are present in the metastore, as well, they are not visible to
```

Comment:
How is the hiding of non-Hudi being done?
This may create some unwanted performance drop while listing tables because the tables need to be checked one by one for format.

Reply:
actually not yet. i'll remove this part.